assets/src/dashboard/components/storyAnimation/test/provider.js

```diff
@@ -141,7 +165,17 @@ describe('StoryAnimation.Provider', () => {
       } = result.current;
 
       getAnimationParts(target).forEach((animationPart, i) => {
-        expect(animationPart).toStrictEqual({ type, args: { ...args[i] } });
+        expect(animationPart).toStrictEqual({
+          type,
+          args: { ...args[i], element: element1 },
+        });
+      });
+
+      getAnimationParts(target2).forEach((animationPart) => {
+        expect(animationPart).toStrictEqual({
+          type,
+          args: { ...args[2], element: element2 },
+        });
```

Comment:
If `elements` are passed in, then each animation part will have access to the `element` associated with it.